osmoutils/accum/accum_test.go

```diff
@@ -284,10 +297,10 @@ func (suite *AccumTestSuite) TestNewPositionCustomAcc() {
 	suite.SetupTest()
 
 	// Setup.
-	accObject := accumPackage.MakeTestAccumulator(suite.store, testNameOne, initialCoinsDenomOne, emptyDec)
+	defaultAccObject := accumPackage.MakeTestAccumulator(suite.store, testNameOne, initialCoinsDenomOne, emptyDec)
```

Comment:
nit: we called the defaultAccObject above an acc object with empty coins, but the defaultAccObject here has coins. Would be nice to stick to a single standard of what "default" is to make tests easier to follow.

Reply:
This is the default per-test. That's the reason why we have a default for every test